tox.ini

```
@@ -1,44 +1,59 @@
 [tox]
-envlist = py27,py3{6,7,8,9,10,11},flake8,docs
-skip_missing_interpreters = true
+envlist = coverage_clean, py27, py38, py3, coverage_report, flake8
```

```suggestion
envlist = coverage_clean, py27, py36, py38, py39, py3, coverage_report, flake8
```
Please keep python 3.6 as that is in RHEL 8 and keep 3.9 for RHEL 9.